version.bzl

```diff
@@ -1,3 +1,3 @@
 # NOTE: If you change the format of this line, you must change the bash command
 # in /scripts/publish to extract the version string correctly.
-DF_VERSION = "1.4.18"
+DF_VERSION = "1.4.21"
```

Comment:
This is definitely a minor version upgrade (We've not been following semver rules very well). 1.5.0